code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm

```diff
@@ -0,0 +1,212 @@
+// Category 2 medicines are medicines that have an ill effect regardless of volume/OD to dissuade doping. Mostly used as emergency chemicals OR to convert damage (and heal a bit in the process). The type is used to prompt borgs that the medicine is harmful.
+/datum/reagent/medicine/C2
+	harmful = TRUE
+
+/******BRUTE******/
+
+/datum/reagent/medicine/C2/arnica
+	name = "Arnica"
+	description = "A unique medicine that heals bruises, scaling with the rate at which one is bleeding out. Constricts blood streams, increasing the amount of blood lost. Overdosing further increases blood loss."
+	color = "#ECEC8D" // rgb: 236	236	141
+	taste_description = "wolf's bane"
+	overdose_threshold = 35
+	reagent_state = SOLID
+
+/datum/reagent/medicine/C2/arnica/on_mob_life(mob/living/carbon/M)
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		if(H.bleed_rate)
+			H.adjustBruteLoss(-0.5*H.bleed_rate*REM)
+			H.bleed_rate += 2
+	..()
+	return TRUE
+
+/datum/reagent/medicine/C2/arnica/overdose_process(mob/living/carbon/M)
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		H.bleed_rate += 2
+	..()
+	return TRUE
+
+/datum/reagent/medicine/C2/acetaminophen
+	name = "Acetaminophen"
+	description = "A common pain reliever. Does minor liver damage."
+	color = "#ECEC8D" // rgb: 236	236	141
+	taste_description = "a common spacehold drug"
+	reagent_state = SOLID
+
+/datum/reagent/medicine/C2/acetaminophen/on_mob_life(mob/living/carbon/M)
+	M.adjustLiverLoss(1*REM)
```

Comment:
Should be M.adjustOrganLoss(ORGAN_SLOT_LIVER, 1*REM). If adjustLiverLoss is still somehow in the game code after I removed it I will be very sad

Reply:
it compiled (tm)

Reply:
adjustLiverLoss is not in the codebase so I think your branch is outdated